app/components/c-add-to-cart.vue

```diff
@@ -0,0 +1,124 @@
+<template>
+  <div :class="b()">
+    <div :class="b('quantity')">
+      <e-input
+        v-model.number="quantity"
+        :min="step"
+        :step="step"
+        type="number"
+        name="quantity"
+      />
+    </div>
+    <div :class="b('button')">
+      <e-button
+        :progress="progress"
+        primary
+        width="full"
+        @click="onClick"
+      >
+        <e-icon :class="b('icon')" icon="i-cart" inline/><span v-if="hasLabel"> {{ $t('c-add-to-cart.addToCart') }}</span>
+      </e-button>
+    </div>
+    <!-- TODO - remove -->
+    <div ref="debug">{{ debug }}</div>
+  </div>
+</template>
+
+<script>
+  import { mapActions } from 'vuex';
+
+  export default {
+    name: 'c-add-to-cart',
+    // components: {},
+    // mixins: [],
+
+    props: {
+      /**
+       * Sku of the article to be added
+       */
+      sku: {
+        type: String,
+        required: true,
+      },
+
+      /**
+       * Increment steps. Also defines initial value
+       */
+      step: {
+        type: Number,
+        default: 1,
+      },
+
+      /**
+       * Show label or not
+       */
+      label: {
+        type: Boolean,
+        default: false,
+      },
+    },
+    data() {
+      return {
+        quantity: this.$props.step,
+        hasLabel: !!this.$props.label,
+        progress: false,
+        debug: '', // TODO - remove
+      };
+    },
+
+    // computed: {},
+    // watch: {},
+
+    // beforeCreate() {},
+    // created() {},
+    // beforeMount() {},
+    // mounted() {},
+    // beforeUpdate() {},
+    // updated() {},
+    // activated() {},
+    // deactivated() {},
+    // beforeDestroy() {},
+    // destroyed() {},
+
+    methods: {
+      ...mapActions({
```

Comment:
- [ ] I would recommend to use "named mappers" as it removes code duplication and gives a better overview of mapped stuff:

```
methods: {
  ...mapActions('cart', [
    'addToCart'
  ])
}
```

See also https://vuex.vuejs.org/en/modules.html#binding-helpers-with-namespace